src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/UpdateReferenceValidation.feature

```diff
@@ -15,11 +15,11 @@ Feature: Update Reference Validation
                   | schoolYear | currentSchoolYear | schoolYearDescription |
                   | 2022       | true              | 2021-2022             |
 
-        
+
         Scenario: 01 Ensure clients cannot update a resource with a Descriptor that does not exist
             Given the system has these "localEducationAgencies" references
-                  | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor                                          | categories                                                                                                                              |
-                  | 10203040               | Institution Test  | "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Federal operated agency | [{ "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Local Education Agency" }]  |
+                  | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor                                           | categories                                                                                                                              |
+                  | 10203040               | Institution Test  | "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Federal operated agency" | [{ "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Local Education Agency" }]  |
```

Comment:
Quotation marks aren't really required on this string. Instead of removing them from all of the descriptor values, I fixed this one missing quotation. It was messing up the color coding of the file in VS Code.